include/ck_tile/ops/gemm/kernel/gemm_kernel.hpp

```diff
@@ -641,7 +640,9 @@ struct GemmKernel
         auto& c_block_window = gemm_tile_windows.at(I2);
 
         EpiloguePipeline{}
-            .template operator()<decltype(c_block_window), decltype(c_block_tile), DstInMemOp>(
+            .template operator()<decltype(c_block_window),
+                                 decltype(c_block_tile),
+                                 EpiloguePipeline::MemoryOperation>(
```

Comment:
You don't have to use it here, since you already have it as an epilogue class type member.